oneapp-server/types/Application.test.js

```diff
@@ -0,0 +1,90 @@
+const { createTestClient, dataSources, services } = require('../__utils/TestingUtils');
+const client = createTestClient();
+const passwordGenerator = require('generate-password');
+
+jest.mock('generate-password');
```

Comment:
We shouldn't need this dependency in the tests. That was in the user tests to mock out the password reset process, but that piece of code shouldn't be touched in these tests.